project_euler/problem_33/sol1.py

```diff
@@ -23,17 +23,17 @@ def isDigitCancelling(num, den):
                 return True
 
 
-def solve(digit_len: int) -> str:
+def solution(digit_len: int) -> str:
```

Comment:
Please add the default value for the parameter `digit_len` such that when called without any arguments `solution()` returns the right answer to the solution.